Capstone/Friends/FriendMatchesCell.h

```diff
@@ -0,0 +1,18 @@
+//
+//  FriendMatchesCell.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 8/1/22.
+//
+#import "ParseCollege.h"
+#import <UIKit/UIKit.h>
+ 
+NS_ASSUME_NONNULL_BEGIN
+ 
+@interface FriendMatchesCell : UITableViewCell
+@property (weak, nonatomic) IBOutlet UIImageView *collegeImage;
```

Comment:
ditto re: naming comments in the FriendsDetailsViewController.h

Reply:
resolved